tests/test_boot.py

```diff
@@ -12,14 +12,22 @@
 
 
 @pytest.mark.integration
-@pytest.mark.parametrize("format", [f for f in OutputFormat if f != OutputFormat.none])
-def test_boot(format: OutputFormat) -> None:
+@pytest.mark.parametrize(
+    "format",
+    [pytest.param(f, marks=pytest.mark.output(f)) for f in OutputFormat if f != OutputFormat.none],
```

Comment:
Hmm, we should probably just run the test for "none" as well but just stop after the build step like the "tar" output.

Reply:
I think I'd leave that to a later PR. I'd like to split `test_boot` apart a bit, but I'd rather get this PR over the line first.